x-pack/legacy/plugins/task_manager/server/queries/mark_available_tasks_as_claimed.ts

```diff
@@ -95,29 +94,9 @@ if (doc['task.runAt'].size()!=0) {
   },
 };
 
-const SORT_VALUE_TO_BE_FIRST = 0;
+// function changed to stop sorting by _id: https://github.com/elastic/kibana/issues/52517
 export const sortByIdsThenByScheduling = (claimTasksById: string[]): SortClause => {
-  const {
-    _script: {
-      script: { source },
-    },
-  } = SortByRunAtAndRetryAt;
-  return defaultsDeep(
-    {
-      _script: {
-        script: {
-          source: `
-if(params.ids.contains(doc['_id'].value)){
-  return ${SORT_VALUE_TO_BE_FIRST};
-}
-${source}
-`,
-          params: { ids: claimTasksById },
-        },
-      },
-    },
-    SortByRunAtAndRetryAt
-  );
+  return SortByRunAtAndRetryAt;
```

Comment:
The reason we sort by ID here is that we want to make sure that tasks that have been queried for by ID are done first as they are called by the `run_now` api.

Removing this will break that assumption and means TM will run these tasks after its finished running all other expired tasks.... so, this shouldn't be removed.